modules/sharedaddy/sharing.js

```diff
@@ -42,8 +42,12 @@ if ( sharing_js_options && sharing_js_options.counts ) {
 						continue;
 					}
 
-					while ( ( service_url = urls[ service ].pop() ) ) {
-						jQuery.getScript( service_url );
+					while ( ( service_request = requests[ service ].pop() ) ) {
```

Comment:
They also introduce an error in `sharing.js` when tested on my Jetpack site:

`Uncaught ReferenceError: requests is not defined`

I think a few lines from the source commit were omitted (specifically, the renaming of some variables). Those should probably be migrated here to simplify future syncing.